src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/user/User.java

```diff
@@ -8,21 +9,83 @@
 /**
  * @author seiferla
  */
-public record User(Address email, String password, String salt) {
+public class User {
```

Comment:
```suggestion
public final class User {
```

Reply:
Entities should have unique IDs. This is what our common package is used for, so that we do not forget

Reply:
I'm not sure here. Shouldn't the e-mail be used as a unique identifier (read: not a numerical id) be used here?

Reply:
You are probably right. Then just please make the class final 👍

Reply:
ay ay sir!